src/gui/PageView.h

```diff
@@ -124,6 +124,16 @@ class XojPageView: public Redrawable, public PageListener {
      * on the display taking into account the current zoom
      */
     int getDisplayHeight() const;
+    /**
+     * Returns the width of this XojPageView as displayed
+     * on the display taking into account the current zoom
+     */
+    double getDisplayWidthDouble() const;
+    /**
+     * Returns the height of this XojPageView as displayed
+     * on the display taking into account the current zoom
+     */
+    double getDisplayHeightDouble() const;
```

Comment:
Since these have the same documentation strings as the methods above them, would it be reasonable to use the same docstring for both? E.g.

```c++
    /**
     * Returns the width of this XojPageView as displayed
     * on the display taking into account the current zoom
     */
    int getDisplayWidth() const;
    double getDisplayWidthDouble() const;
    /**
     * Returns the height of this XojPageView as displayed
     * on the display taking into account the current zoom
     */
    int getDisplayHeight() const;
    double getDisplayHeightDouble() const;
```